docs/src/basics/common_solver_opts.md

```diff
@@ -176,9 +176,9 @@ explanations of the timestepping algorithms, see the
 * `callback`: Specifies a callback. Defaults to a callback function which
   performs the saving routine. For more information, see the
   [Event Handling and Callback Functions manual page](../../features/callback_functions.html).
-* `isoutofdomain`: Specifies a function `isoutofdomain(t,u)` where, when it
+* `isoutofdomain`: Specifies a function `isoutofdomain(u,t)` where, when it
```

Comment:
u,p,t. It's seen here: https://github.com/JuliaDiffEq/DiffEqBase.jl/blob/master/src/common_defaults.jl